src/main/java/org/jitsi/videobridge/rest/StatisticsRequestHandler.java

```diff
@@ -0,0 +1,284 @@
+     * @throws IOException
+     */
+    void handleStatsRequest(
+            String target,
+            HttpServletRequest request,
+            HttpServletResponse response)
+            throws IOException
+    {
+        if ("GET".equals(request.getMethod()))
+        {
+            JSONObject statsJsonObject = new JSONObject();
+            if (target.equals(STATISTICS))
+            {
+                statsJsonObject = doGetStatisticsJSON();
+            }
+            else if (target.equals(STATISTICS + "/" + NODE))
+            {
+                // TODO
+                statsJsonObject = new JSONObject();
+                //statsJsonObject.put(
+                //        "node_stats", StatsKeepingNode.Companion.getStatsMap());
+            }
+            else if (target.equals(STATISTICS + "/" + POOL))
+            {
+                // TODO
+                //statsJsonObject = ByteBufferPool.getStatsJson();
+            }
+            else if (target.equals(STATISTICS + "/" + QUEUE))
+            {
+                // TODO
+                //statsJsonObject = QueueStatistics.getStatsJson();
+            }
+            else if (target.equals(STATISTICS + "/" + TRANSIT))
+            {
+                // TODO
+                //statsJsonObject = TransitTime.getStatsJson();
+            }
+            else
+            {
+                response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+                return;
+            }
+
+
+            response.setStatus(HttpServletResponse.SC_OK);
+            statsJsonObject.writeJSONString(response.getWriter());
+        }
+        else if ("POST".equals(request.getMethod()))
+        {
+            if (!RESTUtil.isJSONContentType(request.getContentType()))
+            {
+                response.setStatus(HttpServletResponse.SC_UNSUPPORTED_MEDIA_TYPE);
+                return;
+            }
+
+            JSONObject requestJSONObject;
+            try
+            {
+                Object o = new JSONParser().parse(request.getReader());
+                if (o instanceof JSONObject)
+                {
+                    requestJSONObject = (JSONObject) o;
+                }
+                else
+                {
+                    requestJSONObject = null;
+                }
+            }
+            catch (Exception e)
+            {
+                requestJSONObject = null;
+            }
+
+            if (requestJSONObject == null)
+            {
+                response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                return;
+            }
+
+            Boolean all = getBoolean(requestJSONObject.get(ALL));
+            Boolean node = getBoolean(requestJSONObject.get(NODE));
+            Boolean queue = getBoolean(requestJSONObject.get(QUEUE));
+            Boolean pool = getBoolean(requestJSONObject.get(POOL));
+            Boolean transit = getBoolean(requestJSONObject.get(TRANSIT));
+
+            if (node != null || (node = all) != null)
```

Comment:
what does the url look like to enable/disable stats?

Reply:
Example: 
POST
{ "all" : false, "pool": true}
To disable everything and enable the pool stats.